### PR TITLE
fix: allow non-string pruning

### PR DESCRIPTION
**Pull request by saartochner**

### CODE REVIEW

File: src/utils/payloadStringify.test.js [hunk 1]
```diff
@@ -116,6 +116,14 @@ describe('payloadStringify', () => {
     expect(result).toEqual('[2,3,3,4,5,6,7,11]...[too long]');
   });
 
+  test('prune on non-string', () => {
```
Comment (OrrLumigo) [inline src/utils/payloadStringify.test.js:119]: would add undefined and null tests

**Review by OrrLumigo — CHANGES_REQUESTED** [review]

File: src/utils/payloadStringify.js [hunk 2]
```diff
@@ -30,7 +31,14 @@ export const keyToOmitRegexes = () => {
   return regexesList.map((x) => new RegExp(x, 'i'));
 };
 
-export const prune = (str, maxLength) => (str || '').substr(0, maxLength);
+export const prune = (str, maxLength) => {
```
Comment (doriaviram) [inline src/utils/payloadStringify.js:34]:
```suggestion
?
```

**Review by doriaviram — CHANGES_REQUESTED** [review]

File: src/utils/payloadStringify.js (hunk 2 above)
Comment (saartochner) [reply src/utils/payloadStringify.js:34]: didn't get this comment 🤔

**Review by OrrLumigo — APPROVED** [review]

**Review by doriaviram — APPROVED** [review]
Please find the root cause :)

**Pull request merged**